plutus-tx-plugin/test/Plugin/Data/Spec.hs

```diff
@@ -69,8 +69,8 @@ instance P.Eq MyMonoData where
     (Mono3 i1) == (Mono3 i2)       = i1 P.== i2
```

Comment:
Perhaps add a coment in all of these that says that we have to patternmatch, so we don't optimize away the `data => type`?

Reply:
:+1: